appuio/redis/values.yaml

```
@@ -611,62 +588,53 @@ slave:
     ##
     # nodePort:
 
-    ## Provide any additional annotations which may be required. This can be used to
-    ## set the LoadBalancer service type to internal only.
+    # -- Provide any additional annotations which may be required. 
+    ## This can be used to set the LoadBalancer service type to internal only.
     ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
-    ##
     annotations: {}
     labels: {}
     loadBalancerIP:
     # loadBalancerSourceRanges: ["10.0.0.0/8"]
 
-  ## Redis(TM) slave port
-  ##
+  # -- Redis(TM) slave port
   port: 6379
-  ## Deployment pod host aliases
+  # -- Deployment pod host aliases
   ## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
-  ##
   hostAliases: []
-  ## Can be used to specify command line arguments, for example:
+  # -- Can be used to specify command line arguments
   ## Note `exec` is prepended to command
-  ##
   command: "/run.sh"
-  ## Additional commands to run prior to starting Redis(TM)
-  ##
+  # -- Additional commands to run prior to starting Redis(TM)
   preExecCmds: ""
-  ## Additional Redis(TM) configuration for the slave nodes
+  # -- Additional Redis(TM) configuration for the slave nodes
   ## ref: https://redis.io/topics/config
-  ##
   configmap:
-  ## Redis(TM) extra flags
-  ##
+  # -- Redis(TM) extra flags
   extraFlags: []
-  ## List of Redis(TM) commands to disable
-  ##
+  # -- List of Redis(TM) commands to disable
   disableCommands:
     - FLUSHDB
     - FLUSHALL
 
-  ## Redis(TM) Slave pod/node affinity/anti-affinity
-  ##
+  # -- Redis(TM) Slave pod/node affinity/anti-affinity
   affinity: {}
 
-  ## Kubernetes Spread Constraints for pod assignment
+  # -- Kubernetes Spread Constraints for pod assignment
   ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
   ##
   # - maxSkew: 1
   #   topologyKey: node
   #   whenUnsatisfiable: DoNotSchedule
```

This gets rendered, looking strange
```suggestion
  ## - maxSkew: 1
  ##   topologyKey: node
  ##   whenUnsatisfiable: DoNotSchedule
```